charts/osm/grafana/dashboards/osm-mesh-envoy-details.json

```diff
@@ -435,13 +434,19 @@
           "interval": "",
           "legendFormat": "Total N pods in Mesh",
           "refId": "A"
+        },
+        {
+          "expr": "osm_proxy_connect_count",
+          "interval": "",
+          "legendFormat": "OSM Proxy Connect",
+          "refId": "B"
         }
       ],
       "thresholds": [],
       "timeFrom": null,
       "timeRegions": [],
       "timeShift": null,
-      "title": "Total number of pods in Mesh",
+      "title": "Pods in Mesh vs Connected Proxies",
```

Comment:
This title might not make much sense to users, is there something else we can put in here?

Reply:
Let me revert this one. I struggled to give it a name; I think you are right.